Equipment.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <iostream>
+#include "LeakedObjectDetector.h"
+#include "Laundromat.h"
+#include "WashingMachine.h"
+#include "Restaurant.h"
+#include "Server.h"
+#include "Speaker.h"
+
+struct Equipment
+{
+    Laundromat machine;
+    WashingMachine pipe;
+    Restaurant dishWasher;
+    Server carrier;
+    Speakers tweaters;
+    int heavyMachines = this->machine.numWashers + this->machine.numDryers;
```

Comment:
a better name for this variable is `numHeavyMachines` or `heavyMachineCount`
Name things what they are or what they do
- [ ] revise